pkgs/tools/filesystems/fatresize/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, parted, substituteAll, utillinux, ... }:
+
+stdenv.mkDerivation rec {
+  
+  version = "1.0.2";
+  name = "fatresize-${version}";
+
+  src = fetchurl {
+    url = mirror://sourceforge/fatresize/fatresize-1.0.2.tar.bz2;
+    sha256 = "04wp48jpdvq4nn0dgbw5za07g842rnxlh9mig4mslz70zqs0izjm";
+  };
+  
+  phases = [
```

Comment:
Remove `phases` section. As I can see you have `preConfigure` not mentioned here.